source/manual/deploying-terraform.html.md

```diff
@@ -23,65 +23,77 @@ to our AWS environments.
 
 You can find which class of user you are [infra-security project in govuk-aws-data](https://github.com/alphagov/govuk-aws-data/tree/master/data/infra-security).
 
-## Deploying manually on Jenkins
+### 2. Install `gds-cli`
 
-> **This is not the recommended approach.**
->
-> It's slightly easier to start with, but you'll quickly want to switch to using the `deploy.rb` helper script (see next section), to avoid pasting secrets over and again!
+[`gds-cli`](/manual/access-aws-console.html) is the preferred way of obtaining
+AWS credentials.
 
-### 1. Generate an AWS token and key
+As of version `v2.15.0` of [`gds-cli`](/manual/access-aws-console.html),
```

Comment:
You link to the same place as the line above - I'd suggest just removing this link altogether.